cypress/integration/goToTeamPage.spec.ts

```diff
@@ -1,13 +1,16 @@
+// TODO: do not use { force: true } option
+// This option was added as a workaround for test scenarios failing due to "the center of this element is hidden from view"
+// Related discussion on repo: https://github.com/eddieantonio/predictive-text-studio/pull/293#issuecomment-817975467
 describe("Go to the team page", function () {
   it("should reach the team page via a link on the landing page", function () {
     cy.visit("/");
-    cy.data("team-page-link").contains("About the Team").click();
+    cy.data("team-page-link").contains("About the Team").click({ force: true });
     cy.url().should("include", "/team");
   });
 
   it("should see loaded profiles on the team page", function () {
     cy.visit("/team");
-    cy.wait(500);
+    cy.wait(1000);
```

Comment:
I had to adjust this value to make the test passing, but this could possibly be with my network connection. Please let me know if 500 still works in other machines.